scheduler/scheduler_system.go

```diff
@@ -251,6 +253,15 @@ func (s *SystemScheduler) computeJobAllocs() error {
 		s.plan.AppendStoppedAlloc(e.Alloc, allocLost, structs.AllocClientStatusLost, "")
 	}
 
+	for _, e := range diff.disconnecting {
+		s.plan.AppendUnknownAlloc(e.Alloc)
+	}
+
+	// Log reconnect updates. They will be pulled by the client when it reconnects.
+	for _, e := range diff.reconnecting {
+		s.logger.Trace("reconnecting alloc", "alloc_id", e.Alloc.ID, "alloc_modify_index", e.Alloc.AllocModifyIndex)
+	}
```

Comment:
Because the scheduler is in the hot path, we typically only log "top-level" issues. Is this super valuable given that the data all lands in the plan output and given that we can't guarantee this is actually valid until the plan applier runs over it?

Reply:
It's useful but mostly valuable for debugging. I had meant to ask if that directive stood for TRACE too.  Now I know!